packages/terra-application-links/tests/jest/ApplicationTabs.test.jsx

```diff
@@ -11,6 +11,11 @@ it('should render ApplicationTabs with links and alignment', () => {
   expect(wrapper).toMatchSnapshot();
 });
 
+it('should render ApplicationTabs with links and alignment and onTabClick handler', () => {
+  const wrapper = enzyme.shallow(<div><ApplicationTabs links={testLinkConfig} alignment="start" onTabClick={() => {}} /></div>);
+  expect(wrapper).toMatchSnapshot();
```

Comment:
Can we have a test that onTabClick is called when clicking the tab and avoid using snapshots? Here is an example: https://github.com/cerner/terra-framework/blob/7a086d9c163b297c985e420867a3042b870dec62/packages/terra-file-path/tests/jest/FilePath.test.jsx#L71